website/content/docs/api-clients/api/pagination.mdx

```diff
@@ -0,0 +1,129 @@
+
+To request the next page of results, run the `list` operation again and include the `list_token` that Boundary provided as a parameter.
+Boundary returns the next page of results with new `response_type` and `list_token` values.
+
+![API pagination workflow](/img/api-pagination/pagination-workflow.png)
+
+You can continue viewing new pages of results using the `list_token` values until the `response_type` reads `complete`.
+
+## Refresh the local cache
+
+When the `response_type` reads `complete`, you have viewed all the results that were available when you ran the initial `list` operation.
+You can now refresh the cached list of results to return any resources that were updated since you started viewing the list responses.
+
+To refresh the cached list results, make another `list` request using the `list_token` from the previous response.
+Boundary returns a `response_type` and `list_token` as part of the response, similarly to when you ran the initial operation.
+If the `response_type` is `delta`, it means there are more items available in the list.
+You can use the `list_token` as a parameter to continue viewing new pages of results until the `response_type` reads `complete`.
+
+Boundary also includes the top-level response `deleted_ids` as part of the refresh operation.
+The `deleted_ids` response contains the IDs of any resources that were deleted since the initial `list` operation.
+Deleted resources are included in the results of the list for 30 days after they are deleted.
+Resources that have been deleted include the suffix `_deleted`.
+
+After 30 days, the `list_token` expires and you must make a new `list` request without the token to regenerate the full list of results.
+
+## Response structure
+
+All `list` endpoints support the following parameters:
+
+- `list_token` (optional) - An opaque token that is returned from the previous `list` response.
+If you do not supply a value, pagination starts from the beginning.
+- `page_size` (optional) - An unsigned integer that indicates the number of items that should be included on the page of search results.
+If you do not include a value, or if you include a value of `0`, Boundary uses the default page size of 1000 items.
+If you configure it, the `page_size` value overrides the default page size.
+Controller administrators can also configure a `max_page_size` option that constrains the maximum page size a user can request.
+
+Boundary returns the following response parameters:
+
+- `response_type` - Defines whether the response is a `delta` or `complete` listing.
+The value is `delta` until the last page is returned, at which point it will be `complete`.
+- `list_token` - An opaque token which is included in subsequent listing requests after the initial one, so that the client can continue to paginate the results.
+- `sort_by` - The column name that the returned items should be sorted on.
+Each list endpoint has one defined column which defines the sorting for all returned items.
```

Comment:
I don't know that we need this sentence. It'll always be either `created_time` or `updated_time`, but I don't think we need to say that in the docs.